xbmc/windowing/windows/WinEventsWin32.cpp

```diff
@@ -180,7 +182,8 @@ static XBMC_keysym *TranslateKey(WPARAM vkey, UINT scancode, XBMC_keysym *keysym
       keysym->unicode = static_cast<uint16_t>(vkey - VK_NUMPAD0 + '0');
     }
     else if (ToUnicode(static_cast<UINT>(vkey), scancode, keystate,
-                       reinterpret_cast<LPWSTR>(wchars.data()), wchars.size(), 0) > 0)
+                       reinterpret_cast<LPWSTR>(wchars.data()), static_cast<UINT>(wchars.size()),
```

Comment:
```suggestion
                       reinterpret_cast<LPWSTR>(wchars.data()), static_cast<int>(wchars.size()),
```

https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-tounicode

Reply:
you're right, skipped one in the intellisense.